src/main/resources/application.properties

```diff
@@ -72,6 +72,7 @@ challenge26ciphertext=gbU5thfgy8nwzF/qc1Pq59PrJzLB+bfAdTOrx969JZx1CKeG4Sq7v1uUpz
 DEFAULT37=DEFAULT37
 challenge27ciphertext=gYPQPfb0TUgWK630tHCWGwwME6IWtPWA51eU0Qpb9H7/lMlZPdLGZWmYE83YmEDmaEvFr2hX
 challenge41password=UEBzc3dvcmQxMjM=
+DOCKER_SECRET_CHALLENGE49=sdfjklhjUYYkghjv;-jhkf
```

Comment:
Can we maybe encode/encrypt this in order to give it away? we could maybe work together on having a shell script (see https://github.com/OWASP/wrongsecrets/blob/master/.github/scripts/docker-create.sh) to generate the secret in [the](https://github.com/OWASP/wrongsecrets/pull/1611/files#diff-d9c0c07f029efb8a4de5742d0b584dde1cc286a6b76cac7daf24bf781ac89f27) and here. But I am fine with doing this post PR as well :) .

Reply:
You mean generate different secret everytime ??

Reply:
Yes, or just encode the secret here. What would be your take on this @bendehaan ?

Reply:
I would prefer randomization on startup - less noise on current scanners, and lower likelihood contestants in a CTF just find the value in a walkthrough on the web

Reply:
Lets skip the randomization for now :-)